bot/cogs/owo_cog.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import re
+import math
+import random
+
+import discord
+import discord.ext.commands as commands
+
+import bot.extensions as ext
+
+log = logging.getLogger(__name__)
+
+class OwoCog(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.FACES = ["(・`ω´・)", ";;w;;", "owo", "UwU", ">w<", "^w^"]
+
+    @ext.command()
+    @ext.long_help('Owo\'s the text given to the command')
+    @ext.short_help('owo any string')
+    @ext.example('!owo hello there')
+    async def owo(self, ctx):
+
+        message = ctx.message.content
+        message = message.replace(ctx.prefix + ctx.invoked_with + ' ', '')
```

Comment:
this can be done in a much cleaner fashion with d.pys command framwork arg parsing, see here for an example https://github.com/ClemsonCPSC-Discord/ClemBot/blob/537b1746e566e6e75f60608e1ff6d84ea0b014f6/bot/cogs/eval_cog.py#L37 most notably the ,*, operator referenced here https://discordpy.readthedocs.io/en/latest/ext/commands/commands.html#keyword-only-arguments